packages/scss/src/components/_action-icon.scss

```diff
@@ -64,24 +51,71 @@
 	&.mod-invert {
 		color: _component("actionIcon.invert.color");
 
-		&:hover, &:focus {
-			&:not([disabled]) {
+		&:not([disabled]) {
+			&:hover, &:focus {
 				background-color: rgba(255, 255, 255, .1);
 				color: _component("actionIcon.invert.color");
 			}
+
+			&:active {
+				background-color: rgba(255, 255, 255, .2);
+			}
+		}
+	}
+
+	&.mod-outlined {
```

Comment:
where is the :hover for the mod-outlined (with box-shadow)?

Reply:
Supported by the default version (grey 50 on hover).
But I just noticed the outline also support a light box shadow effect on hover. I'll fix that.